qa-heal.planx-pla.net/dashboard/Public/documentation/index.html

```diff
@@ -1215,6 +1615,34 @@ <h2 id="FAQ" class="section-title">5. FAQ</h2>
                         If errors persist, contact our help desk at support@datacommons.io.
                         <br><br>
                     </li>
+                    <li>
+                        <span style="font-weight: bold;">I want to work interactively on the Tutorial Notebooks. Show me how. </span><br><br>
+                        Click <a href="#DemoHEALNotebookBACPAC">here</a> to follow the steps of a demo.
+                        <br><br>
+                    </li>
+                    <li>
+                        <span style="font-weight: bold;">What are the current Tutorial Notebooks?</span><br><br>
+                        Click <a href="#CurrentlyAvailableNotebooks">here</a> to see a list of currently available Notebooks in Python and R.
+                        <br><br>
+                    </li>
+                    <li>
+                        <span style="font-weight: bold;">How do I work with these Tutorial Notebooks?</span><br><br>
+                        Click <a href="#WorkingTutorialNotebooksInteractively">here</a> to 
+                        see a guide of how to work with the Tutorial Notebooks.
+                        <br><br>
+                    </li>
+                    <li>
+                        <span style="font-weight: bold;">How do I link my account to a FAIR enabled repository? </span><br><br>
```

Comment:
No need to change this, but `<br>` for explicit spacing is a bit of a bad smell. It's valid to use a block-level element like `p` inside an `li` if you want to create new-lines. Ideally `bootstrap` or another styling framework helps with simple things like spacing, but this works! 👍

Reply:
I believe the whole page moves to another platform soon..